Remotion/Web/Core/UI/Controls/ListMenuImplementation/Rendering/ListMenuRenderer.cs

```diff
@@ -74,10 +74,31 @@ public void Render (ListMenuRenderingContext renderingContext)
       RegisterMenuItems(renderingContext);
 
       renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Id, renderingContext.Control.ClientID);
-      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassListMenu);
-      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute2.Role, HtmlRoleAttributeValue.Menu);
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, "listMenuRoot");
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute2.Role, "region");
+
       if (IsDiagnosticMetadataRenderingEnabled)
         AddDiagnosticMetadataAttributes(renderingContext);
+
+      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Div);
+
+      var renderHeading = renderingContext.Control.Heading is { IsEmpty: false };
+      if (renderHeading)
+      {
+        var heading = renderingContext.Control.Heading!.Value;
+        var headingID = renderingContext.Control.ClientID + "_Heading";
+        var tag = GetTagFromHeadingLevel(renderingContext.Control.HeadingLevel);
+        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Id, headingID);
+        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, "screenReaderText");
+        renderingContext.Writer.RenderBeginTag(tag);
+        heading.WriteTo(renderingContext.Writer);
+        renderingContext.Writer.RenderEndTag();
+
+        renderingContext.Writer.AddAttribute("aria-labelledby", headingID);
+      }
+
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassListMenu);
```

Comment:
The css class on the table gets dropped and the css is updated to use `div.listMenu table` instead of `table.listMenu`.